Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM python:3.6.5
 
 COPY ./app /flask-explorer/app
 COPY ./server.py /flask-explorer/server.py
+COPY ./models /flask-explorer/models
```

Comment:
the model shouldn't be baked into the image, but we should rather use a volume for that. That way we can just switch out model files

Reply:
yes, indeed!